### ISSUE TITLE
V2.2.0 crashes on startup

### ISSUE THREAD

**Issue by Wakinyantanka** [SoftFever/OrcaSlicer#7290]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

v2.2.0

### Operating System (OS)

Linux

### OS Version

Linux mint

### Printer

Custom BLVMGN Cube

### How to reproduce

Click icon software starts and immediately crashes 

### Actual results

Software crashes

### Expected results

Operational software that doesn't crash on startup

### Project file & Debug log uploads

Cant get log file

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Anything else?

Can't get log file. program isn't open long enough

**Comment by Schakal2102** [comment]
Ab Linux Mint 22, soll die [OrcaSlicer_Linux_Ubuntu2404_V2.2.0.AppImage] verwendet werden.
Bei mir läuft die andere auch nicht, aber diese Version ohne Probleme.

**Comment by Wakinyantanka** [comment]
I'm sorry is there a way to translate this to English?

**Comment by CitrusIntellect** [comment]
Translated bu DuckDuckGo:

Starting with Linux Mint 22, the [OrcaSlicer_Linux_Ubuntu2404_V2.2.0.AppImage] should be used. The other one doesn't work for me either, but this version without any problems.

Let me guess, your using an nvidia card 

Your not alone  #6908 #6520

**Comment by CitrusIntellect** [comment]
#7210

**Comment by Wakinyantanka** [comment]
Yes using nvidia gpu. I tried OrcaSlicer_Linux_Ubuntu2404_V2.2.0.AppImage and it won't even open. Did a fresh install of Mint with the same problems. Please tell me they are working on a fix for this. I do not want to go back to Windoze....

**Comment by kohbo** [comment]
I have this issue and I'm using an AMD 7900xtx

**Comment by kohbo** [comment]
Switching to the latest version on Github 2.2.0 beta 2 is working for me

**Comment by padcom** [comment]
Not working for me when the selected printer is networked - aka the network plugin is enabled an in use. Linux Mint 21 + RTX3060ti if anyone is interested. The issue started happening Today. The network plugin got updated and boom - segfault.

**Comment by klack** [comment]
It appears to be GTK errors for me:
```
(orca-slicer:159484): Gtk-CRITICAL **: 13:53:45.984: gtk_widget_set_size_request: assertion 'width >= -1' failed

(orca-slicer:159484): Gtk-CRITICAL **: 13:53:45.984: gtk_widget_set_size_request: assertion 'width >= -1' failed

(orca-slicer:159484): GLib-GObject-CRITICAL **: 13:53:45.984: invalid cast from 'wxPizza' to 'GtkCellLayout'

(orca-slicer:159484): Gtk-CRITICAL **: 13:53:45.984: gtk_cell_layout_get_cells: assertion 'GTK_IS_CELL_LAYOUT (cell_layout)' failed

(orca-slicer:159484): Gtk-CRITICAL **: 13:53:45.985: gtk_widget_set_size_request: assertion 'width >= -1' failed
```

**Comment by klack** [comment]
Deleting my config folder worked for me.  See:
See https://github.com/SoftFever/OrcaSlicer/issues/6887

**Comment by Wakinyantanka** [comment]
I'll try the beta 2.2.0 when I get home.

**Comment by CitrusIntellect** [comment]
I just compiled OrcaSlicer from nothing and it is working great! I would suggest that first.

**Comment by darco415** [comment]
> I just compiled OrcaSlicer from nothing and it is working great! I would suggest that first.

can you share it?

**Comment by Wakinyantanka** [comment]
> I just compiled OrcaSlicer from nothing and it is working great! I would suggest that first.

What is the process to do that? Thanks!

**Comment by Wakinyantanka** [comment]
I followed the read.me for Ubuntu and get an error that stops the build..

**Comment by darco415** [comment]
> ninja: build stopped: subcommand failed."

same here

**Comment by CitrusIntellect** [comment]
Did you follow instructions for Ubuntu or Linux compilation?

Also if your on Arch Linux and have the aur you can run:
paru -Syu orca-slicer
OR
yay -Syu orca-slicer

Depends on your AUR helper

**Comment by darco415** [comment]
> Deleting my config folder worked for me. See #6887

Ok so I renamed the machine folder, and OS started up fine. Able to upload file to printer

**Comment by Wakinyantanka** [comment]
I purged everything from my machine then installed the Linux appimage,(Ubuntu appimage still won't open at all), rebuilt all my profiles for 6 machines and upload through mainsail. It's the only way I can get it to work. Every time I try to put in the IP address it immediately crashes and will not reopen.

**Comment by Wakinyantanka** [comment]
> Did you follow instructions for Ubuntu or Linux compilation?
> 
> Also if your on Arch Linux and have the aur you can run: paru -Syu orca-slicer OR yay -Syu orca-slicer
> 
> Depends on your AUR helper

I followed The instructions for Ubuntu.

**Comment by darco415** [comment]
> > Did you follow instructions for Ubuntu or Linux compilation?
> > Also if your on Arch Linux and have the aur you can run: paru -Syu orca-slicer OR yay -Syu orca-slicer
> > Depends on your AUR helper
> 
> I followed The instructions for Ubuntu. I'm using Linux Mint Cinnamon 21.3

add the port number 7125 after the ip

**Comment by darco415** [comment]
> I purged everything from my machine then installed the Linux appimage,(Ubuntu appimage still won't open at all), rebuilt all my profiles for 6 machines and upload through mainsail. It's the only way I can get it to work. Every time I try to put in the IP address it immediately crashes and will not reopen.

add the port number 7125 after the ip

**Comment by Wakinyantanka** [comment]
> 
> add the port number 7125 after the ip
Added the port number and I am able to upload directly to my machines without crashing. 
![Screenshot from 2024-11-02 06-14-31](https://github.com/user-attachments/assets/dbce5fd6-9e58-409e-a20f-9510503650d1)
Device tab shows this though.
Still happy to be able to upload directly!!

**Comment by henr7381** [comment]
I assume this is still an issue that is being addressed, but it's not fixed so I figured I'd post this. 

System details:

  Kernel: 5.15.0-130-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Cinnamon 6.0.4
    tk: GTK 3.24.33 wm: muffin vt: 7 dm: LightDM 1.30.0 Distro: Linux Mint 21.3 Virginia
    base: Ubuntu 22.04 jammy

As of 20250107, the dedicated Ubuntu appimage does not launch (to be expected). Furthermore, the appimage crashes when trying to get ahold of the Klipper interface on my printers. I know this, because when I isolate my machine from the subnet with my printers, the appimage launches and runs fine. If I then reactivate the subnet and allow Orcaslicer to try and connect to my printers, it immediately crashes. Putting the Moonraker port after the IP of my printers prevents Orcaslicer from crashing, so there must be an issue with the Klipper interface tab in Orcaslicer.

If you're another user with this problem, as mentioned previously, add `xxx.xxx.xxx.xxx:7125` to the end of your printer's IP to prevent crashes.

**Comment by darco415** [comment]
ya we known this fix since Nov 2
